frontend/src/views/Evaluation/Nomination/NominationTable.tsx

```diff
@@ -16,9 +20,23 @@ interface DataTableRowProps {
 }
 
 const DataTableItemRenderer: React.FC<DataTableRowProps> = ({ item }) => (
-    <ParticipantCard participant={item.participant} />
+    <ParticipantCard key={item.participant.id} participant={item.participant} />
 )
 
+const DeleteColumn: React.FC<DataTableRowProps> = ({ item }) => {
+    if (item.role === Role.Facilitator) return null
```

Comment:
Could we return a react fragment instead? 
```suggestion
    if (item.role === Role.Facilitator) return <></>
```